projects/extension/src/components/Tab.tsx

```diff
@@ -70,7 +69,7 @@ const Tab: FunctionComponent<TabProps> = ({
       /* TODO(nik): Fix smoldot definition (see: https://github.com/paritytech/substrate-connect/blob/3350cdff9c4c294393160189816168a93c983f79/projects/extension/src/background/ConnectionManager.ts#L202)
        ** eslint disable below seems to be due to smoldot definition */
       // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-      manager?.disconnectTab(tab.tabId)
+      disconnectTab(tab.tabId)
```

Comment:
If I understood well - and we want actually yo have multiple apps in each tab - then we should not disconnect the Tab but the app inside the tab.

Reply:
whats an "app"?

Reply:
> whats an "app"?

sorry - I meant networks connections